bus-mapping/src/evm/opcodes/callop.rs

```diff
@@ -101,17 +105,29 @@ impl Opcode for Call {
             state.call_context_write(&mut exec_step, call.call_id, field, value);
         }
 
-        state.transfer(
-            &mut exec_step,
-            call.caller_address,
-            call.address,
-            call.value,
-        )?;
-
         let (_, callee_account) = state.sdb.get_account(&call.address);
         let is_empty_account = callee_account.is_empty();
         let callee_nonce = callee_account.nonce;
         let callee_code_hash = callee_account.code_hash;
+
+        if is_call {
+            state.transfer(
+                &mut exec_step,
+                call.caller_address,
+                call.address,
+                call.value,
+            )?;
+        } else {
```

Comment:
I think for static call, still transfer but with zero value make more like as evm does.

Reply:
Fixed to `transfer` with zero value for opcode `STATICCALL`.